.github/workflows/check_precompiled.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Pass updated files in $updated_files environment variable
+
+# enc_bootloader
+echo "$updated_files" | grep -q "^enc_bootloader/.*\.elf$"
+enc_bootloader_elf_not_updated=$?
```

Comment:
Should this check that _both_ of the enc_bootloader ELFs have been updated? As it is, I think this check would pass if only one of the ELFs was updated but the other wasn't?